frontend/tests/elm-package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0",
+    "summary": "Sample Elm Test",
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD-3-Clause",
+    "source-directories": [
+        ".",
+        "../src/elm"
+    ],
+    "exposed-modules": [],
+    "dependencies": {
+        "elm-community/elm-test": "3.1.0 <= v < 4.0.0",
+        "elm-community/list-extra": "6.0.0 <= v < 7.0.0",
+        "elm-community/json-extra": "2.0.0 <= v < 3.0.0",
+        "elm-lang/core": "5.0.0 <= v < 6.0.0",
+        "elm-lang/html": "2.0.0 <= v < 3.0.0",
+        "elm-lang/http": "1.0.0 <= v < 2.0.0",
+        "elm-lang/navigation": "2.1.0 <= v < 3.0.0",
+        "evancz/url-parser": "2.0.1 <= v < 3.0.0",
+        "mgold/elm-random-pcg": "4.0.2 <= v < 5.0.0",
+        "rtfeldman/node-test-runner": "3.0.0 <= v < 4.0.0"
```

Comment:
Can you elaborate what this library does?
Previous `elm-test` ran without this.